test/functional/terminal/window_spec.lua

```diff
@@ -14,8 +14,17 @@ describe('terminal window', function()
 
   describe('with colorcolumn set', function()
     before_each(function()
-      feed('<c-\\><c-n>:set colorcolumn=20<cr>i')
-      wait()
```

Comment:
I don't think so - the intermediate test makes sure nvim is not in terminal mode, so at the point where nvim is back in terminal mode and the 2nd test possibly could succeed, the colurcolumn command must have been processed.